tests/test_mock_controller.py

```diff
@@ -126,15 +127,15 @@ async def test_moveAz(self):
         self.data = await self.read()
         amcs_status = self.data[Dome.LlcName.AMCS.value]
         self.assertEqual(
-            amcs_status["status"], Dome.LlcStatus.STOPPED.value,
+            amcs_status["status"], Dome.LlcStatus.CRAWLING.value,
         )
-        self.assertEqual(
+        self.assertGreaterEqual(
```

Comment:
Please also test moveAz and moveEl the crawl velocity 0 and negative.

Also I suggest testing elevation near the limits, to make sure it stops when it runs into a limit
and azimuth near the 0/360 boundary to make sure it crosses that boundary without a glitch.

The way I usually do this sort of thing is to write a `check_foo` method that has most of the logic, then call it from one or more `test_foo` methods that provide the conditions you are testing.

Reply:
I introduced test cases that test the 0 and negative velocities as well as moving to higher and lower azimuth values. I also opened a ticket to introduce proper handling of the 0/360 azimuth and 0/90 elevation boundaries:

https://jira.lsstcorp.org/browse/DM-25189